swim/memberlist.go

```diff
@@ -105,20 +113,24 @@ func (m *memberlist) ComputeChecksum() {
 // generates string to use when computing checksum
 func (m *memberlist) GenChecksumString() string {
 	var strings sort.StringSlice
+	var buffer bytes.Buffer
 
 	for _, member := range m.members.list {
```

Comment:
```bash
$ git grep "GenChecksumString"
swim/memberlist.go:     checksum := farm.Fingerprint32([]byte(m.GenChecksumString()))
swim/memberlist.go:func (m *memberlist) GenChecksumString() string {
```

Since it is only called from `ComputeChecksum` I will un-export this function.